js2r-vars.el

```diff
@@ -134,6 +134,7 @@
 
 (defun js2r-rename-var ()
   "Renames the variable on point and all occurrences in its lexical scope."
+	(if (fboundp 'evil-insert-state) (evil-insert-state))
```

Comment:
I don't know evil, but could you check if we're in evil-mode? It would probably a lot safer, because loading and using evil are two different things.